var/spack/repos/builtin/packages/octave/package.py

```diff
@@ -348,6 +351,9 @@ def configure_args(self):
         else:
             config_args.append("--without-z")
 
+        if "~pcre2" in spec:
+            config_args.append("--without-pcre2")
+
```

Comment:
```suggestion
        if spec.satisfies("~pcre2"):
            config_args.append("--without-pcre2")
```

Quick suggestion to switch to `spec.satisfies()` which is recommended over using in spec.